README.md

```diff
@@ -21,7 +21,7 @@ Note: `bodies::ConvexMesh::MeshData` was made implementation-private and is no l
 
 ## Build Status
 
-Travis CI: [![Build Status](https://travis-ci.org/ros-planning/geometric_shapes.svg?branch=melodic-devel)](https://travis-ci.org/ros-planning/geometric_shapes)
```

Comment:
if you update the builds status tags, shouldn't you update/comment out the ones below which refer to ROS1?

Reply:
of course!